docs/getting-started/index.md

```diff
@@ -14,9 +14,9 @@ This guide assumes a basic understanding of working on a Linux based operating
 system using tools like apt-get, npm, yarn, curl. Docker knowledge is also
 helpful for making the best use of your Backstage installation.
 
-If you are planning to contribute plugins or the project in general, we advise
+If you are planning to contribute plugins or to the project in general, we advise
 you to use the
-[Getting Started for Contributors](https://backstage.io/docs/getting-started/running-backstage-locally)
+[Contributors](https://backstage.io/docs/getting-started/contributors)
```

Comment:
again, should use relative so docusaurus generates it...

```suggestion
[Contributors](contributors.md)
```

Reply:
PS: ironically, while this sentence says use the Docs contributors guide, at the BOTTOM of this page it says to use the main one in the root of the repo.  over time something to align to so the info doesn't need to be duplicated. (not sure exactly how much is vs. isn't)

Reply:
Done